internal/service/domain/treecluster/treecluster.go

```diff
@@ -149,8 +141,54 @@ func (s *TreeClusterService) Ready() bool {
 	return s.treeClusterRepo != nil
 }
 
+// handlePrevTreeLocation updates the locations of clusters associated with the provided trees.
```

Comment:
I quickly googled and I think comments with `/* */` are also possible. Maybe that will make it look a little cleaner.

```
/*
 	handlePrevTreeLocation updates the locations of clusters associated with the provided trees.
	This method iterates over a list of trees and processes the clusters they belong to.
	For each cluster, the cluster's location is updated by recalculating its coordinates and region using the `GeoClusterLocator`.
	Clusters are only updated once, even if multiple trees belong to the same cluster.

	Parameters:
		- ctx: The context for managing request-scoped values, cancellation, and timeouts.
		- trees: A slice of Tree entities to process. Each tree may reference a cluster.

	Returns:
		An error if any cluster update fails, otherwise nil.

	Notes:
		- Clusters with an ID of 0 are ignored.
		- Updates are performed via a callback mechanism in the `treeClusterRepo` to ensure thread safety or transactional consistency.
*/

```

Reply:
We should also check which other functions could use some doc comments.

Reply:
Yes, this is my start on adding doc comments. As for the `/* */` thing, I would prefer the `//` style, because even the default lib in go uses this style to document the code.

Example in std http lib
![image](https://github.com/user-attachments/assets/c2d4ad1a-ccda-4ba2-bbac-86cfab6391c3)

Reply:
Hmm … than let's do it the ugly way